tests/test_modeling_tf_common.py

```diff
@@ -2466,6 +2468,7 @@ def test_push_to_hub(self):
                 break
         self.assertTrue(models_equal)
 
+
     def test_push_to_hub_callback(self):
```

Comment:
Because of the new check, which depends on a PT class, this test should get a `@require_torch` decorator... perhaps the new check should be moved to a separate test, due to the extra requirements? 🤔

Reply:
I am adding the `is_pt_tf_cross_test` marker!